index.js

```diff
@@ -25,14 +26,16 @@ module.exports = (input, opts) => {
 	input = input || 'patch';
 
 	opts = Object.assign({
-		cleanup: true
+		cleanup: true,
+		yarn: opts && opts.yarn !== false && hasYarn()
```

Comment:
I think this is wrong. Because if I ran `np --yarn`, this statement will be overridden with `true` and `hasYarn()` has no effect.

It should be

```js
opts = Object.assign({
    cleanup: true
}, opts, {
    yarn: opts && opts.yarn !== false && hasYarn()
});
```